website/content/docs/secrets/mongodbatlas.mdx

```diff
@@ -16,6 +16,8 @@ project or organization with appropriate role(s). The MongoDB Atlas Programmatic
 Private Keys are returned to the caller. To learn more about Programmatic API Keys visit the
 [Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api-docs/apiKeys/).
 
+Please do not confuse this with MongoDB Atlas Database secrets engine described [here](/vault/docs/secrets/databases/mongodbatlas).
```

Comment:
<Note>
  The information below relates to the MongoDB Altas <b>secrets engine</b>. Refer to the
  <a href="vault/docs/secrets/databases/mongodbatlas">MongoDB Atlas database secrets engine</a>
  for information about using the MongoDB Atlas database plugin for the Vault
  database secrets engine.
</Note>

Reply:
Merged the suggestion. Thanks!